_source/logzio_collections/_log-sources/kubernetes.md

```diff
@@ -10,19 +10,32 @@ open-source:
 contributors:
   - idohalevi
   - imnotashrimp
+  - yyyogev
 shipping-tags:
   - container
 ---
 
-## Setup
-
 For Kubernetes, a DaemonSet ensures that some or all nodes run a copy of a pod.
 This implementation is uses a Fluentd DaemonSet to collect Kubernetes logs.
 Fluentd is flexible enough and has the proper plugins to distribute logs to different third parties such as Logz.io.
 
 The logzio-k8s image comes pre-configured for Fluentd to gather all logs from the Kubernetes node environment and append the proper metadata to the logs.
 
-###### Configuration {#configuration}
+<div class="branching-container">
+
+* [Default configuration <span class="sm ital">(recommended)</span>](#default-config)
+* [Custom configuration](#custom-config)
+{: .branching-tabs }
+
+<!-- tab:start -->
+<div id="default-config">
+
+## Deploy logzio-k8s with default configuration
+
+For most environments, we recommend using the default configuration.
+However, you can deploy a [custom configuration](#custom-config) if your environment needs it.
```

Comment:
The link to `custom configuration` doesn't work very good.

Reply:
fixed in the next push